examples/read_bids_datasets.py

```diff
@@ -0,0 +1,183 @@
+"""
```

Comment:
example files that don't start with plot_ are not integrated after running in the doc thanks to sphinx-gallery

Reply:
I think that might be an mne-python specific setting for sphinx gallery that we don't have in mne-bids, see the other example file names: https://github.com/mne-tools/mne-bids/tree/master/examples